src/amp-story-embed.js

```diff
@@ -102,6 +159,37 @@ export class AmpStoryEmbed {
     this.iframeEl_.classList.add('story');
     this.initializeLoadingListeners_();
     this.rootEl_.appendChild(this.iframeEl_);
+    this.initializeHandshake_(story);
+  }
+
+  /**
+   * @param {!Element} story
+   * @private
+   */
+  initializeHandshake_(story) {
+    const frameOrigin = getEncodedUrl(story.href).origin;
+
+    Messaging.waitForHandshakeFromDocument(
+      self,
+      this.iframeEl_.contentWindow,
+      frameOrigin
+    ).then(
+      messaging => {
+        // TODO(Enriqe): Appropiately set visibility to stories.
+        messaging.sendRequest('visibilitychange', {state: 'visible'}, true);
+        messaging.setDefaultHandler(() => {
+          // TODO(Enriqe): Set default handler.
+        });
+        messaging.registerHandler('moreInfoLinkUrl', () => {
+          // TODO(Enriqe): Get publisher defined `moreInfoLinkUrl` (could come
```

Comment:
I don't think we actively need a TODO for this; we could just wait until we've gotten a feature request for this.

Reply:
+1, this whole block (L178-L186) is not needed in this PR. Let's introduce these methods properly with their own PR, `visibilitychange` or `moreInforLinkUrl` should not live in the `initializeHandshake_()` method